src/Token.js

```diff
@@ -23,6 +23,8 @@ export interface LexerInterface {input: string, tokenRegex: RegExp}
 export class Token {
     text: string;
     loc: ?SourceLocation;
+    noexpand: ?boolean; // don't expand the token
+    treatAsRelax: ?boolean; // used in \noexpand
```

Comment:
Is any combination of these valid?

Reply:
It's possible in theory, but there is no use yet.